File: PATTERNS/Pattern47/Pattern47.c
#include <stdio.h>

int main() {
    int i, j, n;
    printf("Enter row number:");
    scanf("%d",&n);
    for (i = 1; i <= n; i++) {
        for (j = n; j >= 1; j--) {
            if (j == i)
                printf("*");
            else
                printf("%d", j);
        }
        printf("\n");
    }
    
    return 0;
}

File: PATTERNS/Pattern47/Pattern47.go
package main

import "fmt"

func main() {
	var n int
	fmt.Print("Enter the row number:")
	fmt.Scanf("%d",&n)
	
	for i:=1; i<=n;i++{
		for j:=n;j>=1;j--{
			if(i==j){
				fmt.Printf("*")
			}else{
				fmt.Printf("%d",j)
			}
		}
		fmt.Println()
	}
}